scripts/globals/status.lua

```diff
@@ -2159,48 +2159,50 @@ dsp.jsa = dsp.jobSpecialAbility;
 dsp.skill =
 {
     -- Combat Skills
-    NON = 0,
-    H2H = 1,
-    DAG = 2,
-    SWD = 3,
-    GSD = 4,
+    NONE = 0,
+    HAND_TO_HAND = 1,
```

Comment:
temped to defend h2h :thinking: but I think Ima take it as it is now

Reply:
If you really want, I don't mind changing it back, it was probably the most descriptive of them all.

Others bordered on being useless, and had to look up what they actually were.